java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobStartedEvent.java

```diff
@@ -44,17 +47,45 @@ public static Builder builder() {
         return new Builder();
     }
 
+    /**
+     * Creates an instance of this class. This constructor is specifically for ingest jobs and is the first event that
+     * happens in an ingest task. This is not used for bulk import jobs, as they are started by being assigned to a
+     * task.
```

Comment:
I've updated the javadoc a bit more, but i feel like maybe we should consider renaming these 2 events? Its a bit difficult trying to explain in the javadoc of `IngestJobStartedEvent` that sometimes it can represent a point which does not start a job run.

Reply:
That might be a good idea. I'm not sure what else we'd call it though.